addon/components/frost-error.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * Component definition for the frost-error component
```

Comment:
I think it would make sense to have the name of this a bit more specific to http error codes.  The way it's structured right now is rather specific and it probably makes more sense to have error presentations for specific cases rather than trying to do one generic "handle everything" version

How about something like `frost-ajax-error-page`?